packages/server/src/utils/S3.ts

```diff
@@ -29,3 +31,28 @@ export const fetchS3Data = (bucket: string, key: string): Promise<string> => {
         })
         .catch(err => Promise.reject(`Failed to fetch S3 object ${bucket}/${key}: ${err}`));
 };
+
+export const streamS3DataByLine = (
```

Comment:
reinstating this function, first used in the [single contributor propensity test](https://github.com/guardian/support-dotcom-components/pull/545/files#diff-c424b987a2a705879411917598854204f0bd691fec513a298bcbbecbea9f9097R32).